backend/config/initializers/form_builder.rb

```diff
@@ -14,7 +14,8 @@ def error_message_on(method, options = {})
 
   def field_hint(method, options = {})
     title = options[:title] || @object.class.human_attribute_name(method)
-    text = options[:text] || I18n.t(method, scope: [:spree, :hints, @object.class.model_name.i18n_key])
+    text_interpolation_args = options.fetch(:text_interpolation_args, {})
+    text = options[:text] || I18n.t(method, scope: [:spree, :hints, @object.class.model_name.i18n_key], **text_interpolation_args)
```

Comment:
I think there may be another way to do this: if we delete the other options we can just pass the remaining ones to the `translate` method instead of using a specific option for this text interpolation. Something like:

```ruby
title = options.delete(:title) || @object.class.human_attribute_name(method)
text = options.delete(:text) || I18n.t(method, scope: [:spree, :hints, @object.class.model_name.i18n_key], **options)
```

The idea is that we could need to pass also other `i18n.t` options along the way (like `default`, for example), and this way it's a bit more flexible, even if less explicit. What do you think?

Reply:
Yeah, agree, I wasn't entirely sure what would be preferred. I will make this change.